### PR TITLE
[docstring] Fix docstring for DonutImageProcessor

### PR DESCRIPTION
**Pull request by abzdel**
fixed DonutImageProcessor docstring and removed from OBJECTS_TO_IGNORE in check_docstrings.py

### CODE REVIEW

**Comment by abzdel** [comment]
@ydshieh should be all set! Happy to make changes if needed

File: src/transformers/models/donut/image_processing_donut.py [hunk 1]
```diff
@@ -76,9 +76,9 @@ class DonutImageProcessor(BaseImageProcessor):
             the `preprocess` method.
         rescale_factor (`int` or `float`, *optional*, defaults to `1/255`):
             Scale factor to use if rescaling the image. Can be overridden by `rescale_factor` in the `preprocess`
-            method.
-        do_normalize:
-            Whether to normalize the image. Can be overridden by `do_normalize` in the `preprocess` method.
+            method. Whether to normalize the image. Can be overridden by `do_normalize` in the `preprocess` method.
+        do_normalize (`bool`, *optional*, defaults to `True`):
+            Whether to normalize the image.
```
Comment (ydshieh) [inline src/transformers/models/donut/image_processing_donut.py:81]: the script has some issue and we have to fix it on our side. But `Whether to normalize the image. Can be overridden by `do_normalize` in the `preprocess` method.` should be after `do_normalize`. Could you fix this manually 🙏 Thanks

**Comment by abzdel** [comment]
@ydshieh Just fixed this with my most recent commit. Just a heads up - I get the following when I run `make fixup`. I looked into fixing it but I wasn't quite able to - happy to keep investigating this if needed
image

**Comment by ydshieh** [comment]
It's probably a TF issue in your environment. Other people also reported this, but I have no clear idea what's going on here as it works for me as well as on CI runner. For this PR, as CI is green, good to merge.

**Review by ydshieh — APPROVED** [review]
Thank you for the contribution 🚀  @abzdel ! Very nice.

**Pull request merged**